samples/sampler/stories/default/ImageItem.js

```diff
@@ -0,0 +1,28 @@
+import ri from '@enact/ui/resolution';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import {text, select} from '@enact/storybook-utils/addons/knobs';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import ImageItem, {ImageItemBase} from '@enact/agate/ImageItem';
+
+ImageItem.displayName = 'ImageItem';
```

Comment:
Hello @Djspaceg 
I'm not sure that we have to define the displayName of the ImageItem here. But other components in agate do it in samplers, too. So is it ok for this code for now?

Reply:
Doing this here helps the code viewer when we are using a component that is wrapped with named HOCs. It only reads the outer-most displayName. It can also help the `Config` object later properly configure the knob grouping, but that's partially handled by the mergeMetadata method.

This is ok, and good practice to do here.